### PR TITLE
Fixed copyright notice

### PR DESCRIPTION
**Pull request by checksum0**
Fixed copyright notice in source file and footer of cashaddress.org.html

### CODE REVIEW

File: cashaddress.org.html [hunk 1]
```diff
@@ -18,14 +19,15 @@
 	*******************		***************
 	window.Crypto			BSD License
 	window.SecureRandom		BSD License
-	window.EllipticCurve		BSD License
+	window.EllipticCurve	BSD License
 	window.BigInteger		BSD License
 	window.QRCode			MIT License
 	window.Bitcoin			MIT License
-	window.Crypto_scrypt		MIT License
+	window.Crypto_scrypt	MIT License
```
Comment (zquestz) [inline cashaddress.org.html]: NIT: INdent doesn't match.

File: cashaddress.org.html [hunk 2]
```diff
@@ -18,14 +19,15 @@
...
 
-	The bitaddress.org software is available under The MIT License (MIT)
+	The cashaddress.org software is available under The MIT License (MIT)
 	Copyright (c) 2011-2016 bitaddress.org (pointbiz)
+	Copyright (c) 2017		cashaddress.org
```
Comment (zquestz) [inline cashaddress.org.html]: NIT: Don't need extra spacing imho.

**Comment by zquestz** [comment]
Just nits, overall looks great. =)

File: cashaddress.org.html (hunk 2 above)
Comment (checksum0) [reply cashaddress.org.html]: Fixed

File: cashaddress.org.html (hunk 1 above)
Comment (checksum0) [reply cashaddress.org.html]: Fixed

**Comment by checksum0** [comment]
Being a nit is good.

**Pull request merged**